dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -1324,7 +1324,7 @@ public Workflow manualRegister(@ApiParam(hidden = true) @Parameter(hidden = true
                             // Store tool table json
                             toolsJSONTable = lInterface.getContent(w.getWorkflowPath(), mainDescriptor.getContent(),
                                     extractDescriptorAndSecondaryFiles(existingTag), LanguageHandlerInterface.Type.TOOLS, toolDAO);
-                            existingTag.setToolTableJson(toolsJSONTable.get());
+                            toolsJSONTable.ifPresent(existingTag::setToolTableJson);
```

Comment:
Did not know you can do this; I like it.

Reply:
I did not know either, Intellij suggested it.